src/BloomBrowserUI/bookEdit/js/bloomEditing.js

```diff
@@ -385,30 +385,82 @@ function MakeSourceTextDivForGroup(group) {
     $(group).each(function () {
       // var targetHeight = Math.max(55, $(this).height()); // This ensures we get at least one line of the source text!
 
-      $(this).qtip({
+      var $this = $(this);
+
+      $this.qtip({
           position: {
-                my: 'left top',
-                at: 'right top',
-              adjust: {
-                  x: 10,
-                  y: 0
-              }
+            my: 'left top',
+            at: 'right top',
+            adjust: {
+              x: 10,
+              y: 0
+            }
           },
           content: $(divForBubble),
 
           show: {
-              event: showEvents,
-              ready: shouldShowAlways
+            event: showEvents,
+            ready: shouldShowAlways
           },
           style: {
-                tip: {
-                    corner: true,
-                    width: 10,
-                    height: 10
-                },
-              classes: 'ui-tooltip-green ui-tooltip-rounded uibloomSourceTextsBubble'
+            tip: {
+              corner: true,
+              width: 10,
+              height: 10
+            },
+            classes: 'ui-tooltip-green ui-tooltip-rounded uibloomSourceTextsBubble'
           },
-          hide: hideEvents
+          hide: hideEvents,
+
+          events: {
+            show: function(event, api) {
+
+              // don't need to do this if there is only one editable area
+              var $body = $('body');
+              if ($body.find("*.bloom-translationGroup").not(".bloom-readOnlyInTranslationMode").length < 2) return;
+
+              // BL-878: set the tool tips to not be larger than the text area so they don't overlap each other
+              var $tip = api.elements.tooltip;
+              var $div = $body.find('[aria-describedby="' + $tip.attr('id') + '"]');
+              var maxHeight = $div.height();
+              if ($tip.height() > maxHeight) {
+
+                // make sure to show a minimum size
+                if (maxHeight < 50) maxHeight = 50;
+
+                $tip.css('max-height', maxHeight);
+                $tip.addClass('qtip-passive');
```

Comment:
I think I'd get confused some day, thinking that "qtip-passive" is something that comes with qtip, when actually we've introduced it?

How about "passive-bubble" instead?